UpSkill_App/UpSkill.Infrastructure/Data/ApplicationDbContext.cs

```diff
@@ -13,14 +13,140 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
 
         public DbSet<Administrator> Administrators { get; init; }
 
-        public DbSet<Owner> Owners { get; init; }
-
-        public DbSet<Employee> Employees { get; init; }
+        public DbSet<ApplicationUser> ApplicationUsers { get; init; }
 
         public DbSet<Coach> Coaches { get; init; }
 
+        public DbSet<Company> Companies { get; init; }
+
         public DbSet<Course> Courses { get; init; }
 
+        public DbSet<Employee> Employees { get; init; }
+
         public DbSet<Grade> Grades { get; init; }
+
+        // TODO:
+        // Can't remember how to make a DBSet of a class that contains T properties
+        //public DbSet<Invoice<???, ???>> Invoices { get; init; }
+
+        public DbSet<LiveSession> LiveSessions { get; init; }
+
+        public DbSet<Owner> Owners { get; init; }
+
+        public DbSet<StudentCourse> StudentCourses { get; init; }
+
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            // <Coach relations>
+            builder
```

Comment:
Not sure if we need all of the fluent apis here but in any case I think cascading delete is something we definitely don't want. :)

Reply:
I was just following the Wiki file, the contents of which we discussed on Saturday.

Reply:
That's what happens when I try to copy paste when I wake up and start working, but my brain is still sleeping 😆